frontend/src/container/GridGraphLayout/Graph/FullView/index.tsx

```diff
@@ -78,7 +88,11 @@ function FullView({
 		[response],
 	);
 
-	if (response.status === 'idle' || response.status === 'loading') {
+	if (
+		response.status === 'idle' ||
+		response.status === 'loading' ||
+		response.isFetching
```

Comment:
no need of `response.isFetching`

Reply:
It is need, this was the reason why UI was getting hanged.